tests/test_integration/test_pipeline.py

```diff
@@ -127,19 +129,18 @@ def test_linear():
     }
     fit = sampling.sample(**linear_input_values)
     samples_test = fit.draws_pd()
-    dict(
-        zip(
-            samples_test.columns,
-            map(list, samples_test.quantile([0.03, 0.97]).T.values),
-        )
+    expected_df = pd.DataFrame.from_dict(
+        expected, orient="index", columns=["5% CI", "95% CI"]
     )
-    assert list(samples_test.columns) == list(expected.keys())
+    samples_test.loc["mean", :] = samples_test.mean()
+    sample_mean = samples_test.loc["mean"].transpose()
+    validation_df = expected_df.join(sample_mean)
```

Comment:
I think it's possible to avoid using a data frame like `validation_df` altogether, but for reference/golf I think these lines could have been done like this:

```
validation_df = expected_df.join(samples_test.mean())
```